src/store/getters/state-getters.ts

```diff
@@ -569,7 +569,8 @@ export const hasPeopleAndRolesChanged = (state: StateIF): boolean => {
 /** Is true when the minimum partners met. */
 export const hasMinimumPartners = (state: StateIF): boolean => {
   // REMOVED Parties are still in the parties array until FILING, so exclude them for component level validations.
-  return getPeopleAndRoles(state).filter(party => !party.actions?.includes(ActionTypes.REMOVED)).length >= 2
+  let isGP = state.resourceModel.entityType === 'GP'
+  return getPeopleAndRoles(state).filter(party => !party.actions?.includes(ActionTypes.REMOVED)).length >= 2 || !isGP
```

Comment:
Use `const` instead of `let` whenever possible (when you are not reassigning the variable).

You can optimize this code by moving `!isGP` first: if it's true then the rest of the "or" expression is not computed.

When an expression has multiple clauses (like here), it's nice if the whole thing is wrapped in parentheses. Sometimes it even looks better on multiple rows, eg,
```
if (
  a ||
  b
)
```